public/assets/css/components/user-management.css

```diff
@@ -90,3 +90,106 @@
   border: 1px solid var(--text-secondary);
   border-radius: 4px;
 }
+
+@media screen and (max-width: 768px) {
+  .admin-container {
+    padding: 1rem;
+    gap: 0.5rem;
+  }
+
+  .admin-control-title {
+    font-size: 1.5rem;
+    text-align: center;
+    margin-bottom: 1rem;
+  }
+
+  .user-search-input {
+    width: 100%;
+    padding: 0.5rem;
+    font-size: 1rem;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    margin-bottom: 1rem;
+  }
+
+  /* Make the table scrollable horizontally */
+  .manage-users {
+    display: block;
+    overflow-x: auto;
+    width: 100%;
+    border-collapse: collapse;
+  }
+
+  /* Ensure each row is clear to read */
+  .manage-users thead {
+    display: none; /* Hide table headers */
+  }
+
+  .manage-users tbody {
+    display: block;
+  }
+
+  .manage-users tr {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid var(--border);
+    margin-bottom: 1rem;
+    padding-bottom: 1rem;
+
+  }
+
+  .users-rows {
+    flex-direction: column;
+    padding: var(--base-padding);
+    & td:last-child {
+          justify-content: center;
+    }
+  }
+
+  .users-rows td {
+    display: flex;
+    justify-content: space-between;
+    padding: 0.5rem 0;
+    font-size: 1rem;
+
+  }
+
+  .users-rows td:first-child {
+    justify-content: center;
+  }
+
+  /* Profile picture adjustments */
+  .profile-pictue {
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+  }
+
+  /* Adjust action buttons and form */
+  .update-status-form {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+  }
+
+  .update-status-form select {
+    width: 100%;
+    padding: 0.5rem;
+    font-size: 1rem;
+  }
+
+  .update-status-form .button {
+    width: 100%;
+    padding: 0.5rem;
+    font-size: 1rem;
+    background: var(--button-primary);
+    color: var(--text-secondary);
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+  }
+
+  .update-status-form .button:hover {
+    background: var(--button-hover);
+  }
+}
```

Comment:
no nesting?

Reply:
It was 4 am, the nesting was being a pain, so I did it without nesting for now

Reply:
What happened to the nesting?

Reply:
fixed